src/pages/Search/SearchTypeMenuNarrow.tsx

```diff
@@ -139,7 +155,7 @@ function SearchTypeMenuNarrow({typeMenuItems, activeItemIndex, queryJSON, title,
         <View style={[styles.pb4, styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween, styles.ph5, styles.gap2]}>
             <PressableWithFeedback
                 accessible
-                accessibilityLabel={popoverMenuItems[activeItemIndex]?.text ?? ''}
+                accessibilityLabel={popoverMenuItems.at(activeItemIndex)?.text ?? ''}
```

Comment:
Please make sure `activeItemIndex` won't be negative since it can affect the logic then.